Lib/difflib.py

```diff
@@ -1038,9 +1038,7 @@ def _qformat(self, aline, bline, atags, btags):
 # remaining is that perhaps it was really the case that " volatile"
 # was inserted after "private".  I can live with that <wink>.
 
-import re
-
-def IS_LINE_JUNK(line, pat=re.compile(r"\s*(?:#\s*)?$").match):
+def IS_LINE_JUNK(line):
```

Comment:
This is public API, we should be careful not to break users.
```suggestion
def IS_LINE_JUNK(line, pat=None):
```

Reply:
Ok, sorry. So should i add this parameter to documentation?

Reply:
No, it just needs to exist at runtime.

Reply:
Done

Reply:
The argument `pat` is now unused. So we could still be breaking users right? 

Removing the `pat` argument makes it more explicit, so I would prefer that.

Either option is fine with me though, in practice I doubt it matters.